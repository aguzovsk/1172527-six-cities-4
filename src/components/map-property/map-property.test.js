import React from 'react';
import renderer from 'react-test-renderer';
import MapProperty from './map-property.jsx';
import offersAmsterdam from '../../test-mock/offers-amsterdam.js';

it(`Map property snapshot test.`, () => {
  const tree = renderer.create(
      (<MapProperty
        offers={offersAmsterdam}
        currentPlace={offersAmsterdam[0]}
      />),
      {
        createNodeMock() {
          return document.createElement(`div`);
        }
      }
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
