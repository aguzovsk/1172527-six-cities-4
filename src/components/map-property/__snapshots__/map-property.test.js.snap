// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map property snapshot test. 1`] = `
<div
  className="map property__map"
  id="map"
  style={
    Object {
      "width": "100%",
    }
  }
/>
`;
